modules/twinklefluff.js

```diff
@@ -348,8 +348,7 @@ Twinkle.fluff.addLinks = {
 };
 
 Twinkle.fluff.disableLinks = function disablelinks(parentNode) {
-	// Array.from not available in IE11 :(
-	$(parentNode).children().each(function(_ix, node) {
+	Array.from($(parentNode).children()).forEach(function (node) {
```

Comment:
I think the intent was to avoid jQuery each() which is confusing as it takes the index before the element. Revereted it now as I see there are lots of jQuery each() uses in the codebase.